static/css/dark-mode.css

```
@@ -0,0 +1,1048 @@
+  background: linear-gradient(135deg, rgba(42, 123, 196, 0.92) 0%, rgba(42, 123, 196, 0.88) 50%, rgba(60, 145, 220, 0.92) 100%);
+  border-color: rgba(255, 255, 255, 0.25);
+  box-shadow: 0 8px 32px rgba(42, 123, 196, 0.5),
+              0 0 0 1px rgba(255, 255, 255, 0.2) inset;
+}
+
+/* ===== 暗色模式：最小化状态 ===== */
+[data-theme="dark"] #chat-container.minimized {
+  background: rgba(42, 123, 196, 0.2);
+}
+
+@keyframes breathingGlowDark {
```

Stylelint (17.3.0)

[error] Expected keyframe name "breathingGlowDark" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)